src/Dropdown/README.md

```diff
@@ -145,7 +145,7 @@ You can use `Dropdown.Toggle` with [IconButton](/components/iconbutton) componen
 ```jsx live
 <Dropdown.Deprecated>
   <Dropdown.Deprecated.Button>
-    <Icon className="fa fa-user pr-3" alt="" />
+    <Icon className="fa fa-user pr-3" />
```

Comment:
[`<Icon>`](https://paragon-openedx.netlify.app/components/icon/#props-api) does not accept an `alt` prop. It accepts an optional `screenReaderText` but it can be omitted for decorative icons, to emulate `alt=""`

Reply:
Good catch!